subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -404,10 +404,13 @@ uint32_t lll_preempt_calc(struct evt_hdr *evt, uint8_t ticker_id,
 	uint32_t diff;
 
 	ticks_now = ticker_ticks_now_get();
-	diff = ticker_ticks_diff_get(ticks_now, ticks_at_event);
+	diff = ticks_now - ticks_at_event;
+	if (diff & BIT(HAL_TICKER_CNTR_MSBIT)) {
```

Comment:
Shouldn't this come after line 412?

Reply:
No, dont want to add `HAL_TICKER_CNTR_CMP_OFFSET_MIN` margin,  and be detected as CPU (ticks_now) being ahead of `ticks_at_event`. But to `return 0` early.